source/declarations.tex

```diff
@@ -905,7 +905,7 @@
 \begin{codeblock}
 constexpr int f(bool b)
   { return b ? throw 0 : 0; }               // OK
-constexpr int f() { return f(true); }       // ill-formed, no diagnostic required
+constexpr int f() { return f(true); }       // ill-formed: no diagnostic required
```

Comment:
This one should stay as-is, and likewise for other "ill-formed, no diagnostic required" cases.